### PR TITLE
Add support Percona Client 8.0.

### PR DESCRIPTION
**Pull request by MemberIT**
Add support Percona Client 8.0
Deps: https://github.com/Oefenweb/ansible-percona-server/pull/33

### CODE REVIEW

**Comment by tersmitten** [comment]
I don't think I understand the https://github.com/Oefenweb/ansible-percona-server/pull/33 dependency. Can you explain it?

**Comment by MemberIT** [comment]
Sorry, I misled you. I meant that changes in this role correspond to changes in Percona Server - no duplicate files are created for apt percona repository, if you use both roles together. You can accept pullrequest earlier than a similar pullrequest in repository Percona Server.

File: vars/main.yml [hunk 1]
```diff
@@ -4,14 +4,28 @@ percona_client_versions_supported:
...
+    url: "http://repo.percona.com/ps-80/apt {{ ansible_lsb.codename }}"
+    component: main
+  - type: deb-src
+    url: "http://repo.percona.com/ps-80/apt {{ ansible_lsb.codename }}"
+    component: main
+
+percona_client_repositories: "{{ percona_client_repositories_8 if percona_client_version == 8.0 else percona_client_repositories_5 }}"
+
+percona_client_version_deb_prefix: "-{{ percona_client_version }}"
+percona_client_version_deb: "{{ '' if percona_client_version == 8.0 else percona_client_version_deb_prefix }}"
```
Comment (tersmitten) [inline vars/main.yml]: Let's use the `version` filter here

**Comment by tersmitten** [comment]
We might want to use `include_vars` the avoid the inline `if`'s extra `variables` and make it more feature proof. What do you think?

**Comment by MemberIT** [comment]
> We might want to use `include_vars` the avoid the inline `if`'s extra `variables` and make it more feature proof. What do you think?

I think it is a good idea. Do this for all ansible percona repositories?

**Comment by MemberIT** [comment]
@tersmitten, do you want me to rewrite the role using `include_vars`?

**Pull request merged**